src/map/script.c

```diff
@@ -9688,10 +9688,23 @@ BUILDIN(setgroupid) {
 
 /// Returns the group ID of the player.
 ///
-/// getgroupid() -> <int>
+/// getgroupid({<account ID>/<character ID>/<character name>})
```

Comment:
here it should just take GID (account ID). if you just know the name at this point you can turn it into id with `getcharid(CHAR_ID_ACCOUNT, "player name")`